echidna/core/dsextract.py

```diff
@@ -73,27 +73,80 @@ class EnergyExtractMC(Extractor):
       fv_radius (float, optional): Fiducial radius. Applies a cut to remove
         events which have a radial position greater than the radius of the
         fiducial volume. If None no cut is applied.
+      reco_pos (bool, optional): If true then position cuts will be made on
+        reconstructed position. If False (default) then MC position is used
+        for cuts.
+
+    Attributes:
+      _reco_pos (bool): If true then position cuts will be made on
+      reconstructed position. If False then MC position is used for cuts.
     '''
 
-    def __init__(self, fv_radius=None):
+    def __init__(self, fv_radius=None, reco_pos=False):
         '''Initialise the class
         '''
         super(EnergyExtractMC, self).__init__("energy_mc", fv_radius)
+        self._reco_pos = reco_pos
+
+    def fv_cut_ntuple(self, entry):
+        """ Applies the fiducial volume (fv) cut to an ntuple entry.
+
+        Args:
+          entry (:class:`ROOT.TChain`): chain entry
```

Comment:
One more minor point on docs, missing `Returns:` info on some of these e.g. something like:
```
Returns:
  bool: Pass/fail FV cut
```
here.